src/cljs/ataru/hakija/subs.cljs

```diff
@@ -123,6 +123,13 @@
   :application/hakukohde-query
   (fn [db _] (get-in db [:application :hakukohde-query])))
 
+(re-frame/reg-sub
+  :application/show-more-hakukohdes?
+  (fn [db _]
+    (let [hakukohdes-visible          (-> db :application :hakukohde-hits count)
```

Comment:
Tän pitäis varmaankin olla `(not-empty (-> db :application :remaining-hakukohde-search-results))`?